backend/de.metas.handlingunits.base/src/main/java/org/adempiere/mm/attributes/api/impl/SubProducerAttributeDAO.java

```diff
@@ -43,8 +44,8 @@
 	{
 		final IQuery<I_C_BP_Relation> bpRelationQuery = Services.get(IQueryBL.class)
 				.createQueryBuilder(I_C_BP_Relation.class, ctx, ITrx.TRXNAME_None)
-				.addEqualsFilter(org.compiere.model.I_C_BP_Relation.COLUMNNAME_C_BPartnerRelation_ID, bpartnerId)
-				.addEqualsFilter(I_C_BP_Relation.COLUMNNAME_IsMainProducer, true)
+				.addEqualsFilter(org.compiere.model.I_C_BP_Relation.COLUMNNAME_C_BPartner_ID, bpartnerId)
```

Comment:
is this OK or it's a typo?
(i.e. chainging C_BPartnerRelation_ID with C_BPartner_ID)

Reply:
I think it was a typo before